libmscore/chord.cpp

```diff
@@ -2285,11 +2285,9 @@ void Chord::layoutTablature()
                   }
             }
       if (!tab->genDurations()            // if tab is not set for duration symbols
-            || track2voice(track())       // or not in first voice
-            || isGrace()) {               // or chord is a grace (no dur. symbols for graces
```

Comment:
Yes, Falckenhagen uses this quite often. And I think it is better to have it (and hide it eventually = stemless) than not having it and to be urged to use grafical things or other workarounds.
Grace notes are only used after Weiss, but sometimes there are (before Weiss) pseudo-grace-notes, that show the exact note for an appogiatura (then without a duration symbol).
Probably it would be better to use another (new) variable to change that for certain scores, but I would not do that for 2.1, as this again would be a new thing, that is not necessary. But we could implement that for 3.0, what do you think?